docs/gfm-alerts-to-admonitions/tests/test_plugin.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from gfm_alerts_to_admonitions import gfm_alert_to_admonition
+from markdown_it import MarkdownIt
+
+
+@pytest.mark.parametrize("kind", ["tip", "note", "important", "warning", "caution"])
+def test_parse(data_regression, kind):
+    input = f"> [!{kind.upper()}]\n> This is an *alert*"
```

Comment:
`upper` shouldn't be needed, GFM-alerts are case-insensitive.

Maybe we should test mixed cases too?

Reply:
The regex _should_ match different cases, we could indeed add a test

Reply:
The implementation looks good, we probably don't need a test but it can't hurt.